backend/src/main/java/fk/prof/backend/model/request/RecordedProfileIndexes.java

```diff
@@ -28,8 +28,8 @@ public void update(Recorder.IndexedData indexedData) {
   private void updateMethodIndex(List<Recorder.MethodInfo> methods) {
     if (methods != null) {
       for (Recorder.MethodInfo methodInfo : methods) {
-        methodLookup.put(methodInfo.getMethodId(), String.format("%s.%s%s",
-            methodInfo.getClassFqdn(), methodInfo.getMethodName(), methodInfo.getSignature()));
+        methodLookup.put(methodInfo.getMethodId(),
+            methodInfo.getClassFqdn() + "." + methodInfo.getMethodName() + methodInfo.getSignature());
```

Comment:
May be we should have a space between methodName and signature and perhaps we should use # as separator between method-name and fqdn.

Eg. 

java.lang.String.getBytes(Ljava.nio.charset.Charset)[B  (which is not very readable)

becomes

java.lang.String#getBytes (Ljava.nio.charset.Charset)[B

which is easier to read.